event_routing_backends/backends/events_router.py

```diff
@@ -154,8 +163,15 @@ def send(self, event):
         Arguments:
             event (dict): the original event dictionary
         """
-        event_routes = self.prepare_to_send([event])
+        if settings.EVENT_ROUTING_BACKEND_BATCHING_ENABLED:
+            events = self.queue_event(event)
+            if not events:
+                return
 
+            self.bulk_send(events)
```

Comment:
I think we want to catch errors here and throw events back on the queue when that happens. Otherwise we're just endlessly reading from the queue and losing them during a downtime.

Reply:
Ready